pkg/instance/instance.go

```diff
@@ -21,6 +23,11 @@ import (
 	"github.com/longhorn/longhorn-instance-manager/pkg/types"
 )
 
+const (
+	maxMonitorRetryCount = 10
+	pollInterval         = 1 * time.Second
+)
```

Comment:
Suggest renaming `pollInterval` to `monitorRetryPollInterval`

Reply:
Updated. Thank you.